### ISSUE TITLE
New registrations not working

### ISSUE THREAD

**Issue by davidnicholls125** [johnvanbreda/naturespot-d8#33]
We have received several reports this morning of the registration form giving an error message when submitted. I've just tried and found the same. I think it has been like this for a couple of days from the reports. Please could you investigate? Thanks.

**Comment by johnvanbreda** [comment]
This should be sorted - the recaptcha v3 module was not properly installed.

**Comment by davidnicholls125** [comment]
Thanks John. It has worked since you added recaptcha 3 so I wonder what has changed in the last few days?

**Comment by johnvanbreda** [comment]
I've realised why. I did an upgrade the other day which overwrote some changes applied for recaptcha_v3. Unfortunately recaptcha_v3 relies on something called Composer for installation which we've not used for our other modules - this changes a folder that gets overwritten when you upgrade, unless you use Composer for the upgrade itself. That's a bit of a pain because Composer has to be run on my local machine rather than on the server, so an upgrade would require me to backup and restore the site locally, upgrade it, then backup and restore to the server. 

Before the next upgrade I'll need to think about whether this is the best approach, or if we can find a way around this, or perhaps an alternative module.
